Matrices/C++/Tridiagonal_matrices.cpp

```diff
@@ -0,0 +1,42 @@
+
+//C++ program for Tridiagonal matrix
+#include <iostream>
+using namespace std;
+
+void main()
+{
+int i,j,k;
+int mat[5][5]={{1,2,3,4,5},{6,7,8,9,10},{11,12,13,14,15},{16,17,18,19,20},{21,22,23,24,25}};
```

Comment:
Basic idea is to create a tridiagonal matrix
Ask the user to input the order of matrix .
Then accordingly ask the elements required for tridiagonal matrix to be inputted by the user . And later print the output in matrix form